keylime/README.md

```diff
@@ -1,4 +1,47 @@
 # Keylime Demo Files
 
 This folder contains the various files needed for Keylime, such as configuration
-autorun scripts and payload / revocation scripts
+autorun scripts and payload / revocation scripts, whitelists and exclude lists.
+
```

Comment:
Maybe include the overall Architecture image here, to clarify what part's of keylime are running where